workflows/amplicon/qiime2/qiime2-I-import/QIIME2-Id-demultiplexed-data-paired-end.ga

```diff
@@ -193,15 +198,15 @@
                     "output_name": "imported_data"
                 }
             },
-            "tool_id": "toolshed.g2.bx.psu.edu/repos/q2d2/qiime2_core__tools__import/qiime2_core__tools__import/2023.5.0+dist.h193f7cc9.3",
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/q2d2/qiime2_core__tools__import/qiime2_core__tools__import/2024.10.0+dist.h3d8a7e27",
             "tool_shed_repository": {
-                "changeset_revision": "10447a4b0cc3",
+                "changeset_revision": "8a0db69428c7",
                 "name": "qiime2_core__tools__import",
                 "owner": "q2d2",
                 "tool_shed": "toolshed.g2.bx.psu.edu"
             },
-            "tool_state": "{\"import_root\": {\"type\": \"SampleData__ob__PairedEndSequencesWithQuality__cb__\", \"__current_case__\": 46, \"__q2galaxy__GUI__cond__format__\": {\"format\": \"CasavaOneEightLanelessPerSampleDirFmt\", \"__current_case__\": 0, \"import_sequences\": {\"__q2galaxy__GUI__cond__sequences__\": {\"__q2galaxy__GUI__select__picker__\": \"collection\", \"__current_case__\": 0, \"elements\": {\"__class__\": \"ConnectedValue\"}, \"__q2galaxy__GUI__cond__add_ext__\": {\"__q2galaxy__GUI__select__ext_pick__\": \"no\", \"__current_case__\": 1}}}}}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
-            "tool_version": "2023.5.0+dist.h193f7cc9.3",
+            "tool_state": "{\"import_root\": {\"type\": \"SampleData__ob__PairedEndSequencesWithQuality__cb__\", \"__current_case__\": 100, \"__q2galaxy__GUI__cond__format__\": {\"format\": \"CasavaOneEightLanelessPerSampleDirFmt\", \"__current_case__\": 0, \"import_sequences\": {\"__q2galaxy__GUI__cond__sequences__\": {\"__q2galaxy__GUI__select__picker__\": \"collection\", \"__current_case__\": 0, \"elements\": {\"__class__\": \"ConnectedValue\"}, \"__q2galaxy__GUI__cond__add_ext__\": {\"__q2galaxy__GUI__select__ext_pick__\": \"no\", \"__current_case__\": 1}}}}}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
```

Comment:
`SampleData__ob__PairedEndSequencesWithQuality__cb__` this has changed from 46 to 100, does that need to go to the changelog ?

Reply:
Seems that the dict entries are:

- `\"type\": \"SampleData__ob__PairedEndSequencesWithQuality__cb__\"`
- `\"__current_case__\": 46`
- ...

No idea what current case is? Is it just the index of the selected element of this select? Then it should be fine. Othwise I have no idea what `current_case` is, there is no such variable: https://github.com/qiime2/galaxy-tools/blob/main/tools/suite_qiime2_core__tools/qiime2_core__tools__import.xml

Reply:
Ah, right, ok then